bliss/encoder.py

```diff
@@ -295,11 +293,13 @@ def _get_loss(self, pred: Dict[str, Distribution], true_tile_cat: TileCatalog):
         gal_bands = [self.GAL_FLUX_NAMES[band] for band in self.bands]
         for i, (star_name, gal_name) in enumerate(zip(star_bands, gal_bands)):
             # star flux loss
+            star_fluxes[star_fluxes == 0] = 1e-18
```

Comment:
Probably better to do `star_fluxes = star_fluxes.clamp(min=1e-18)`, so we don't keep flux values in (0,1e-18).

Reply:
Also, is there a way to do this substitution in the DC2 data loader, rather than here? It's only need for DC2 presumably.